__tests__/Unit/Components/Tasks/TaskDependency.test.tsx

```diff
@@ -0,0 +1,130 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import TaskDependency from '../../../../src/components/taskDetails/TaskDependency';
+
+describe('TaskDependency', () => {
+    const dependencyDataMock: (
+        | PromiseFulfilledResult<{ title: string | undefined; id: string }>
```

Comment:
We can re-use the type defined in the `taskDetailsApi.ts`

Reply:
done